language/move-prover/stackless-bytecode-generator/src/lifetime_analysis.rs

```diff
@@ -3,7 +3,10 @@
 
 use std::collections::{BTreeMap, BTreeSet};
 
-use bytecode_verifier::absint::{AbstractDomain, JoinResult};
+use bytecode_verifier::{
```

Comment:
Ideally we would remove those trivial deps as well.

Reply:
Removed.